.lychee.toml

```diff
@@ -11,6 +11,9 @@ exclude = [
 
     # GitHub user smibarber referenced in `CREDITS.md` no longer exist
     '^https://github.com/smibarber',
+
+    # OSDev has added bot protection and accesses my result in 403 Forbidden.
+    '^https://wiki.osdev.org',
```

Comment:
No need to split for this time. Generally we want to have distinct changes from separate PRs.  Please keep this in mind in the future. :)